axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -2692,4 +2698,15 @@ InvoiceSet.length > 0))"
       method="showSupplierInvoiceLines"/>
   </action-method>
 
+  <action-method name="action-invoice-method-get-invoice-product-statement">
+    <call class="com.axelor.apps.account.web.InvoiceController"
+      method="getInvoiceProductStatement"/>
+  </action-method>
```

Comment:
I disagree with this implementation, please remove the action method call where it is not necessary (actually I think the action-method can be completely removed) and instead make the change in java code so existing services will call the new InvoiceProductStatementService.